.gitignore

```diff
@@ -0,0 +1,4 @@
+/project_depatment/.idea
+/project_depatment/project_depatment/__pycache__/
```

Comment:
you excluded only /project_depatment/project_depatment/__pycache__/ and /project_depatment/.idea, please use wildcards to make it more general and exclude all possible pycache and .idea directories from the project.

Reply:
fixed